tests/unit/components/supportWithdraw/SupportProcess.spec.ts

```diff
@@ -5,6 +5,7 @@ import appStore from '../../../../src/store'
 import AppModule from '../../../../src/vuexModules/AppModule'
 import Web3Module from '../../../../src/vuexModules/Web3Module'
 import SupportWithdrawModule from '../../../../src/vuexModules/SupportWithdrawModule'
+import PurchaseModule from '../../../../src/vuexModules/PurchaseModule'
```

Comment:
why are there changes to `SupportProcess.spec.ts` at all?  Please remove *all* of these changes.

Reply:
so apparently we need these for travis.  le sigh.